.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: java
+
+jdk:
+  - openjdk8
```

Comment:
Are we sure we want to support all these versions?

Reply:
After doing more research we only want to support the versions designated as Long-Term-Support (LTS) releases. This would mean Java 8 and 11. Oracle designates an LTS release every 3 years. Even though Java 14 is out it is not an LTS and so we don't have to worry about supporting it. The next LTS should be released in ~2021